CMakeLists.txt

```diff
@@ -39,11 +39,7 @@ EXEC_PROGRAM(
 )
 
 IF (SC_GLOBAL_PYTHON_WRAPPER)
-	IF (APPLE)
```

Comment:
Can you confirm that ```/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7/Python.h``` exists and that /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 is not in the default include search path? According to https://github.com/SeisComP3/seiscomp3/blob/master/src/system/libs/swig/CMakeLists.txt#L1 we do set the Python include directory.